qa/tasks/s3tests.py

```diff
@@ -432,6 +432,8 @@ def run_tests(ctx, config):
         attrs = ["!fails_on_rgw", "!lifecycle_expiration", "!fails_strict_rfc2616","!test_of_sts","!webidentity_test"]
         if client_config.get('calling-format') != 'ordinary':
             attrs += ['!fails_with_subdomain']
+        if client_config.get('without-sse-s3'):
+            attrs += ['!sse-s3']
```

Comment:
> problems in s3tests w/o vault - sse-s3 tests are run that probably shouldn't be

@mdw-at-linuxbox in this teuthology piece, we filter out the s3test cases tagged `sse-s3` when we're not configured for it. in s3tests, we can use the decorator `@attr('sse-s3')` to mark these. just make sure your new test cases have that too

Reply:
@mdw-at-linuxbox did this comment get resolved?